test/test-parser.js

```diff
@@ -531,6 +531,7 @@ bodystructure:
                       'wed,': 16,
                       dec: 2015,
                       '16:19:25': '+0100',
+                      CET: ' "Subjec',
```

Comment:
this is the test result when processing a malformed body structure, which now returns an extra line.